app/actions/EventActions.ts

```diff
@@ -249,7 +249,7 @@ export function unregister({
 export function adminRegister(
   eventId: number,
   userId: number,
-  poolId?: number,
+  poolId: number | undefined,
```

Comment:
I'm more fan of the `?` as an optional indicator. I assume you changed this because ts doesn't allow required parameters to follow an optional parameter, but then I'd prefer to just move `poolId?: number,` down the list.

Reply:
Yeah I do agree, but it's used in this order elsewhere and I didn't want to change that here. Sure it's a little uglier but it's not that big of a deal IMO